hub-templates/daskhub/values.yaml

```diff
@@ -41,7 +41,6 @@ basehub:
               - port: 22
                 protocol: TCP
       cloudMetadata:
-        enabled: true
```

Comment:
And this is now `true` by default, correct?

Reply:
No it renamed to the other field below called blockWithIpTables or something, and inverted its value. Having this set leads to an error as the helm charts forces a migration. There is no actual change in outcome by this change now.

Reply:
OK, thanks for the additional info!